libcxx/test/std/utilities/function.objects/refwrap/refwrap.const/type_conv_ctor.pass.cpp

```diff
@@ -57,10 +59,7 @@ int main(int, char**)
   {
     meow(0);
   }
-  {
-    extern std::reference_wrapper<int> purr();
-    ASSERT_SAME_TYPE(decltype(true ? purr() : 0), int);
-  }
+  { ASSERT_SAME_TYPE(decltype(true ? purr() : 0), int); }
```

Comment:
```suggestion
  ASSERT_SAME_TYPE(decltype(true ? purr() : 0), int);
```
Maybe the same for meow while you're at it.

Reply:
Can do!